inlong-sort/sort-common/src/main/java/org/apache/inlong/sort/protocol/node/load/DorisLoadNode.java

```diff
@@ -0,0 +1,118 @@
+ * doris load node using doris flink-doris-connector-1.13.5_2.11
+ */
+@EqualsAndHashCode(callSuper = true)
+@JsonTypeName("dorisLoadNode")
+@Data
+@NoArgsConstructor
+public class DorisLoadNode extends LoadNode implements Serializable {
+
+    private static final long serialVersionUID = -8002903269814211382L;
+
+    @JsonProperty("fenodes")
+    @Nonnull
+    private String feNodes;
+
+    @JsonProperty("username")
+    @Nonnull
+    private String userName;
+
+    @JsonProperty("password")
+    @Nonnull
+    private String password;
+
+    @JsonProperty("table.identifier")
+    @Nonnull
+    private String tableIdentifier;
+
+    @JsonProperty("primaryKey")
+    private String primaryKey;
+
+    @JsonCreator
+    public DorisLoadNode(@JsonProperty("id") String id,
+                         @JsonProperty("name") String name,
+                         @JsonProperty("fields") List<FieldInfo> fields,
+                         @JsonProperty("fieldRelations") List<FieldRelation> fieldRelations,
+                         @JsonProperty("filters") List<FilterFunction> filters,
+                         @JsonProperty("filterStrategy") FilterStrategy filterStrategy,
+                         @Nullable @JsonProperty("sinkParallelism") Integer sinkParallelism,
+                         @JsonProperty("properties") Map<String, String> properties,
+                         @Nonnull @JsonProperty("fenodes") String feNodes,
+                         @Nonnull @JsonProperty("username") String userName,
+                         @Nonnull @JsonProperty("password") String password,
+                         @Nonnull @JsonProperty("table.identifier") String tableIdentifier,
+                         @JsonProperty("primaryKey") String primaryKey) {
+        super(id, name, fields, fieldRelations, filters, filterStrategy, sinkParallelism, properties);
+        this.feNodes = Preconditions.checkNotNull(feNodes, "fe nodes is null");
+        this.userName = Preconditions.checkNotNull(userName, "username is null");
+        this.password = Preconditions.checkNotNull(password, "password is null");
+        this.tableIdentifier = Preconditions.checkNotNull(tableIdentifier, "table.identifier is null");
+        this.primaryKey = primaryKey;
+    }
+
+    @Override
+    public Map<String, String> tableOptions() {
+        Map<String, String> options = super.tableOptions();
+        options.put(DorisConstant.CONNECTOR, "doris");
+        options.put(DorisConstant.FE_NODES, feNodes);
+        options.put(DorisConstant.USERNAME, userName);
+        options.put(DorisConstant.PASSWORD, password);
+        options.put(DorisConstant.TABLE_IDENTIFIER, tableIdentifier);
+        return options;
+    }
+
+    @Override
+    public String genTableName() {
+        return String.format("table_%s", super.getId());
+    }
+
+    @Override
+    public String getPrimaryKey() {
+        return super.getPrimaryKey();
+    }
+
+    @Override
+    public List<FieldInfo> getPartitionFields() {
```

Comment:
It is recommend to remove it  because of no change between it and the supper class.

Reply:
ok. i have fixed it